### PR TITLE
fix: issues for building stencil locally

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: packages/db-ui-elements-stencil/tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "jsx": "react",
     "jsxFactory": "h",
     "esModuleInterop": true,
-    "allowJs": true
+    "allowJs": true,
+    "skipLibCheck": true
```
Comment (mfranzke) [inline packages/db-ui-elements-stencil/tsconfig.json:17]: @nmerget what's the reason for this configuration?

Comment (nmerget) [reply packages/db-ui-elements-stencil/tsconfig.json:17]: https://www.typescriptlang.org/tsconfig#skipLibCheck

There was an issue with @types/jest using some older types from node<16

Comment (mfranzke) [reply packages/db-ui-elements-stencil/tsconfig.json:17]: @nmerget when does this problem come up, like e.g. during which npm script?

**Comment by nmerget** [comment]
We don't need this anymore

**Pull request closed without merging**